src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -364,6 +366,23 @@ export default class ResultsViewOncoprint extends React.Component<
         ) as ClinicalTrackConfigMap;
     }
 
+    @computed get selectedGeneticTrackConfig(): GeneticTrackConfigMap {
+        let geneticTracks: GeneticTrackConfig[] | undefined = this.props.store
+            .pageUserSession.userSettings?.geneticlist;
+        if (geneticTracks) {
+            const userSettingsTracksMap = {} as GeneticTrackConfigMap;
```

Comment:
minor quibble but try to use funtional approach:

const userSettingsTrackMap = genetTracks.map(...)